<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Websites</class>
 <widget class="QDialog" name="Add_Websites">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>264</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>264</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Auto Savior : Add WebSites</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/src.qrc">
    <normaloff>:/pic/Icon.png</normaloff>:/pic/Icon.png</iconset>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>331</width>
     <height>251</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="title">
      <property name="maximumSize">
       <size>
        <width>329</width>
        <height>23</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #ff0000;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;ADD WebSites&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="maximumSize">
       <size>
        <width>329</width>
        <height>3</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="maximumSize">
       <size>
        <width>329</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>16</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="web_site_4">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #0000ff;</string>
          </property>
          <property name="text">
           <string>Web Site Url</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="web_site_name_4">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #0000ff;</string>
          </property>
          <property name="text">
           <string>Web Site Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="login_4">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #0000ff;</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pwd_4">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #0000ff;</string>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="cPwd_4">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #0000ff;</string>
          </property>
          <property name="text">
           <string>Confirm password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="email_4">
          <property name="maximumSize">
           <size>
            <width>86</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #0000ff;</string>
          </property>
          <property name="text">
           <string>Email</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLineEdit" name="url_line_4">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>www.example.com</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="web_site_name_line_4">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>example</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="login_line_4">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pwd_line_4">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>AutoSavior</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="c_pwd_line_4">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>AutoSavior</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="email_line_4">
          <property name="maximumSize">
           <size>
            <width>231</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>example@example.com</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="maximumSize">
       <size>
        <width>329</width>
        <height>5</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>35</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="maximumSize">
         <size>
          <width>165</width>
          <height>23</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="add">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #0000ff;</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>23</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #0000ff;</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="maximumSize">
       <size>
        <width>329</width>
        <height>3</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resource/src.qrc"/>
 </resources>
 <connections/>
</ui>
